apps/admin_app/lib/admin_app_web/mailer/user_email.ex

```diff
@@ -0,0 +1,20 @@
+defmodule AdminAppWeb.Email do
+  @moduledoc """
+  Composing email to send forgot password token.
+  """
+
+  use Phoenix.Swoosh, view: AdminAppWeb.EmailView, layout: {AdminAppWeb.LayoutView, :email}
+  import Swoosh.Email
+  alias AdminAppWeb.Mailer
+
+  def password_reset_mail(token, email, base_url) do
+    sender_email = Application.fetch_env!(:admin_app, :sender_email)
+
+    new()
+    |> to(email)
+    |> from({"Snitch", sender_email})
+    |> subject("Update your password")
+    |> render_body("password_reset_email.html", %{token: token, base_url: base_url})
+    |> Mailer.deliver()
```

Comment:
This code looks cleaner. Please use the same in `session_controller` as well. 
Try to create methods that can be piped instead of calling on in another function